source/libraries/CompileSolidity.ts

```diff
@@ -24,12 +25,12 @@ class SolidityContractCompiler {
     public async compileContracts(): Promise<CompileContractsOutput> {
         try {
             // Compile all contracts in the specified input directory
-            const inputJson: CompilerInput = await this.generateCompilerInput();
-            const compilerOutput: any = compileStandardWrapper(JSON.stringify(inputJson), this.readCallback);
-            const compileOutputJson = JSON.parse(compilerOutput);
-            if (compileOutputJson.errors) {
+            const compilerInputJson: CompilerInput = await this.generateCompilerInput();
+            const compilerOutput: any = compileStandardWrapper(JSON.stringify(compilerInputJson), this.readCallback);
```

Comment:
Recommend removing the `any` here. `compileStandarWrapper` should be returning a non-any type (`CompilerOutput`?).  If it isn't, we should add that.  In general avoiding `any` types is a good idea as it gives you compile time errors (which are easy to fix) rather than runtime exceptions.

Reply:
Yeah, we don't have CompilerOutput defined, but I'll work on adding that and let you know if I have questions.